net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -109,11 +118,15 @@ function arpwatch_resync_config_command() {
 }
 
 function arpwatch_get_arp_file($ifname) {
-	return ARPWATCH_LOCAL_DIR."/arp_$ifname.dat";
+	return ARPWATCH_LOCAL_DIR."/arp_{$ifname}.dat";
 }
 
-function arpwatch_update_vendors() {
-	download_file(ARPWATCH_ETHERCODES_URL, ARPWATCH_LOCAL_DIR."/ethercodes.dat");
+function arpwatch_update_vendors($args) {
+	// download_file(ARPWATCH_ETHERCODES_URL, ARPWATCH_LOCAL_DIR."/ethercodes.dat");
+	$temp = '/tmp/oui.csv';
```

Comment:
Is there any chance this code can run more than one instance simultaneosly? If yes it's better to use a temporary name for temp file

Reply:
Let's eliminate the temp file issues and just pipe fetch to the python script.